AEPAssurance/Source/AssuranceSessionOrchestrator.swift

```diff
@@ -24,68 +40,114 @@ class AssuranceSessionOrchestrator: AssurancePresentationDelegate {
 
     init(stateManager: AssuranceStateManager) {
         self.stateManager = stateManager
+        outboundEventBuffer = ThreadSafeArray(identifier: "Session Orchestrator's OutboundBuffer array")
     }
 
-    func createSession() {
+    /// Creates and starts a new  `AssuranceSession` with the provided `AssuranceSessionDetails`.
+    ///
+    /// A new AssuranceSession is only created if there isn't an already existing session.
+    /// Calling this method also shares the shared state for the extension with the provided session details
+    ///
+    /// - Parameters:
+    ///    - sessionDetails: An `AssuranceSessionDetails` instance containing all the essential data for starting a session
+    func createSession(withDetails sessionDetails: AssuranceSessionDetails) {
+        if let _ = session {
+            Log.warning(label: AssuranceConstants.LOG_TAG, "An active session already exists. Cannot create a new one.")
```

Comment:
Should this be a trace?  I don't think the app developer has much control over this getting invoked more than once, so the warning seems a log-level too high.

Reply:
This gets invoked and log is printed when the user attempts to scan a griffon deeplink on an already connected session. 
Thought it might right to throw a warning to let the user know that their deeplink scan is voided

I have changed the log message to be very specific to the user